src/Compiler/Microsoft.Net.Compilers.Razor.Toolset/build/Microsoft.Net.Compilers.Razor.Toolset.targets

```diff
@@ -0,0 +1,20 @@
+<Project>
+  <!-- Run after https://github.com/dotnet/sdk/blob/main/src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.SourceGenerators.targets#L15 -->
+  <Target Name="_PrepareRazorSourceGeneratorsToolset"
+    BeforeTargets="GenerateMSBuildEditorConfigFileShouldRun"
+    DependsOnTargets="_PrepareRazorSourceGenerators">
+
+    <ItemGroup>
+      <Analyzer Remove="@(_RazorAnalyzer)" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <_RazorSdkSourceGeneratorDirectoryRootOverride>$(MSBuildThisFileDirectory)..\source-generators\</_RazorSdkSourceGeneratorDirectoryRootOverride>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_RazorAnalyzerOverride Include="$(_RazorSdkSourceGeneratorDirectoryRootOverride)*.dll" />
+      <Analyzer Include="@(_RazorAnalyzerOverride)" />
```

Comment:
What is the advantage in having two different item groups here vs just doing the *.dll in the second one?

Reply:
Likely none technically, I just wanted these things in imperative "how I was thinking of them" order.